deployments/rpm/deployment.cfg

```diff
@@ -27,6 +27,17 @@ VERAISON_GROUP=${VERAISON_GROUP:-veraison}
 # vts-service), and a rootCA.crt that was used to sign the service certs.
 VERAISON_CERTS=${VERAISON_CERTS:-}
 
-VERAISON_ROOT=/usr/local/veraison/
+VERAISON_BIN_DIR=/usr/bin
+VERAISON_CERTS_DIR=/etc/veraison/certs
+VERAISON_CONFIG_DIR=/etc/veraison/config
+VERAISON_ENV_DIR=/etc/veraison/env
+VERAISON_LAUNCHD_DIR=/Library/LaunchDaemons #ToDo: double-check this for Darwin
```

Comment:
this not releveant for  `rpm` and should not be set here

Reply:
ACK